internal/k8s/service/resolver.go

```diff
@@ -246,7 +246,7 @@ func (r *backendResolver) consulServiceForK8SService(ctx context.Context, namesp
 			return err
 		}
 		if resolved == nil {
-			return NewConsulResolutionError("consul service not found")
+			return NewConsulResolutionError(fmt.Sprintf("consul service %s not found", namespacedName))
```

Comment:
@andrewstucki preference on using the namespaced name here vs `(<namespace>, <name>)` as we do in some other places?

Reply:
Is namespaced name the `ns/service-name` style? I guess my preference would be towards consistency, but no opinion on specific style.

Reply:
@mikemorris This will print as `<namespace>/<service-name>`, yes. The other format I mentioned is only used in one place, but I suppose that's as good a convention as any if you don't feel strongly about it.

Reply:
Ah, that's kinda weird mixing two platforms, and better explains why it may be structured that way - I couldn't find anything obvious about if `ns/svc` is a convention ever used in Kubernetes - your proposed syntax here LGTM.

Reply:
Reverted to the original syntax